### PR TITLE
template-fonts

### PR DESCRIPTION
**Pull request by narekhovhannisyan**
- added fonts resources from which it's possible to get fonts by id and set character size
 - added getFonts getter and setFonts, resetFonts setters to project data instance
 - added usage examples for resources and getter/setters
 - updated the documentation according to changes and updated old mistakes
 - release 0.4.2

### CODE REVIEW

File: docs/FONTS.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+[See example](/samples/fonts/get-template-available-fonts.js)
+
+Function returns object containing all fonts. The object which returned has method `getFontById` which accepts `fontId` 
+as parameter and returns flatten font. Also given flatten font'a `character size` can be modified by calling 
+`setCharacterSize` which accepts font size as parameter.
+
+```js
+const primaryFont = fonts.getFontById(256).setCharacterSize(20) // returns flatten font with tuned character size
```
Comment (ArtyomHov) [inline docs/FONTS.md]: Users can't set character size property.

Comment (emarukyan) [reply docs/FONTS.md]: users can set font SCALE !

File: docs/project-data-api/PROJECT_DATA_API.md [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
   - [Get Project-data](#get-project-data)
+  - [Update Project-data](#update-project-data)
     * [Getters](#getters)
+      - [Get project id](#get-project-id)
+      - [Get template id](#get-template-id)
+      - [Check if project is Equalizer](#check-if-project-is-equalizer)
+      - [Check if project is Lego](#check-if-project-is-lego)
+      - [Get project title](#get-project-title)
+      - [Get mute music](#get-mute-music)
+      - [Get mute music](#get-mute-music)
```
Comment (ArtyomHov) [inline docs/project-data-api/PROJECT_DATA_API.md]: Duplicated line.

File: docs/FONTS.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Renderforest.getTemplateAvailableFonts(1021)
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+
+[See example](/samples/fonts/get-template-available-fonts.js)
+
+Function returns object containing all fonts. The object which returned has method `getFontById` which accepts `fontId` 
+as parameter and returns flatten font. Also given flatten font'a `character size` can be modified by calling 
+`setCharacterSize` which accepts font size as parameter.
```
Comment (ArtyomHov) [inline docs/FONTS.md]: Please revise the description. It's not understandable.

File: lib/resources/fonts.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+            return this
+          }
+        }
+      }
+
+      return fonts
+    })
+}
+
```
Comment (ArtyomHov) [inline lib/resources/fonts.js]: remove setCharacterSize functionality.

Comment (narekhovhannisyan) [reply lib/resources/fonts.js]: չէք ուզում այսինքն? թասքի մեջ կա

File: docs/FONTS.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  .then(console.log) // handle the success
+  .catch(console.error) // handle the error
+```
+
+[See example](/samples/fonts/get-template-available-fonts.js)
+
+`getTemplateVailabeFonts` function returns object containing all fonts. 
+The object which returned has method `getFontById` which accepts `fontId` as parameter.
+`getFontId` method return flatten font.
+
```
Comment (ArtyomHov) [inline docs/FONTS.md]: Please write this way:
`getTemplateAvailabeFonts ` function returns an object containing all fonts. The returned object has `getFontById ` method which accepts `fontId` parameter. 
`getFontId` method returns flattened font.

Comment (ArtyomHov) [reply docs/FONTS.md]: it is not understandable it return an id of single font or all ids of available fonts.

Comment (narekhovhannisyan) [reply docs/FONTS.md]: Let's))

File: docs/FONTS.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+**Be careful**: `setFonts()` project data instance setter accepts only flatten font. 
+
```
Comment (ArtyomHov) [inline docs/FONTS.md]: flattened not flatten.

Comment (narekhovhannisyan) [reply docs/FONTS.md]: done

File: docs/project-data-api/PROJECT_DATA_API.md [hunk 7]
```diff
@@ -1,9 +1,31 @@
...
+      - [Get screen](#get-screen)
+      - [Get screens](#get-screens)
     * [Setters](#setters)
-    * [Push Screen](#push-screen)
+       - [Set mute music](#set-mute-music)
+       - [Set project styles](#set-project-styles)
+       - [Set project colors](#set-project-colors)
+       - [Set project voice over](#set-project-voice-over)
+       - [Set project sounds](#set-project-sounds)
+       - [Set project colors](#set-project-colors)
```
Comment (ArtyomHov) [inline docs/project-data-api/PROJECT_DATA_API.md]: [Set project colors](#set-project-colors)
Duplicated line.

Comment (narekhovhannisyan) [reply docs/project-data-api/PROJECT_DATA_API.md]: solved

**Pull request merged**